packages/taquito-michelson-encoder/src/schema/error.ts

```diff
@@ -20,3 +20,86 @@ export class InvalidScriptError extends Error {
     super(message);
   }
 }
+
+/**
+ *  @category Error
+ *  @description Error that indicates an invalid RPC response being passed or used
+ */
+export class InvalidRpcResponseError extends Error {
+  public name = 'InvalidRpcResponseError';
+  constructor(public script: any) {
+    super(`
+      Invalid RPC response passed as argument(s): \n
+      ${JSON.stringify(script)}
```

Comment:
Showing the script in the error message can make the error message very long. It is better to not include it in the error message?

Reply:
that's a good point. I will remove it but keep the property